src/Files.App/Helpers/GitHelpers.cs

```diff
@@ -37,11 +37,9 @@ internal static class GitHelpers
 		private static readonly PullOptions _pullOptions = new();
 
 		private static readonly string _clientId =
-#if STORE || STABLE || PREVIEW
-			"githubclientid.secret"
-#else
-			string.Empty;
-#endif
+			EnvHelpers.GetAppEnvironmentAndLogo().Item1 is AppEnvironment.Store or AppEnvironment.Stable or AppEnvironment.Preview
+				? "githubclientid.secret"
```

Comment:
How should I generate the secret?

Reply:
https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app